docs/tutorial/index.rst

```diff
@@ -0,0 +1,270 @@
+(or the one of your choosing).
+This is the repository you will import on Read the Docs,
+and it contains the following files:
+
+``README.rst``
+  Basic description of the repository, you will leave it untouched.
+
+``pyproject.toml``
+  Python project metadata that makes it installable.
+  Useful for automatic documentation generation from sources.
+
+``lumache.py``
+  Source code of the fictional Python library.
+
+``docs/``
+  Directory holding all the Sphinx documentation sources,
+  including some required dependencies in ``docs/requirements.txt``,
+  the Sphinx configuration ``docs/source/conf.py``,
+  and the root document ``docs/source/index.rst`` written in reStructuredText.
+
+Sign up for Read the Docs
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To sign up for a Read the Docs account,
+navigate to the `Sign Up page <https://readthedocs.org/accounts/signup/>`_
+and choose the option "Sign up with GitHub".
+On the authorization page, click the green "Authorize readthedocs" button.
+
+.. figure:: /_static/images/tutorial/github-authorization.png
+   :width: 60%
+   :align: center
+   :alt: GitHub authorization page
+
+   GitHub authorization page
+
+.. note::
+
+   Read the Docs needs elevated permissions to perform certain operations
+   that ensure that the workflow is as smooth as possible,
+   like installing webhooks.
+   If you want to learn more,
+   check out :ref:`connected-accounts:permissions for connected accounts`.
+
+After that, you will be redirected to Read the Docs,
+where you will need to confirm your e-mail and username.
+Clicking the "Sign Up »" button will create your account
+and redirect you to your *dashboard*.
+
+By now, you should have two email notifications:
+
+* One from GitHub, telling you that "A third-party OAuth application ...
+  was recently authorized to access your account". You don't need to do
+  anything about it.
+* Another one from Read the Docs, prompting you to "verify your email
+  address". Click on the link to finalize the process.
+
+Finally, you created your account on Read the Docs
+and are ready to import your first project.
+
+Welcome!
+
+.. figure:: /_static/images/tutorial/rtd-empty-dashboard.png
+   :width: 80%
+   :align: center
+   :alt: Read the Docs empty dashboard
+
+   Read the Docs empty dashboard
+
+.. note::
+
+   Our commercial site offers some extra features,
+   like support for private projects.
+   You can learn more about :doc:`our two different sites </choosing-a-site>`.
+
+First steps
+-----------
+
+Importing the project to Read the Docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To import your GitHub project to Read the Docs,
+first click on the "Import a Project" button on your dashboard
+(or browse to `the import page <https://readthedocs.org/dashboard/import/>`_ directly).
+You should see your GitHub account under the "Filter repositories" list on the right.
+If the list of repositories is empty, click the "refreshing your accounts" link,
```

Comment:
Probably another place to use the actual button image or copy. I _think_ this will work. 

```suggestion
If the list of repositories is empty, click the |:arrows_counterclockwise:| button,
```